create-logux-creator/index.d.ts

```diff
@@ -197,8 +202,42 @@ export type LoguxReduxOptions = ClientOptions & {
   cleanEvery?: number
 }
 
+/**
+ * Connects Logux Client to Redux createStore function.
+ *
+ * ```js
+ * import { CrossTabClient, createStoreCreator } from '@logux/redux'
+ *
+ * const client = new CrossTabClient({
+ *   subprotocol: '1.0.0',
+ *   server: process.env.NODE_ENV === 'development'
+ *     ? 'ws://localhost:31337'
+ *     : 'wss://logux.example.com',
+ *   userId: userId.content
+ *   token: token.content
+ * })
+ *
+ * const createStore = createStoreCreator(client)
+ *
+ * const store = createStore(reducer)
+ * store.client.start()
+ * ```
+ *
+ * @param client Logux Client.
+ * @param options Logux Redux options.
+ * @returns Redux’s `createStore` compatible function.
+ */
+export function createStoreCreator<
+  C extends Client = Client<{}, Log<ClientMeta>>,
+  L extends Log = Log<ClientMeta>
+> (
+  client: Client | CrossTabClient,
```

Comment:
Can we automatically set `C` from `client` argument? Maybe this code will do it (need to be tested):

```suggestion
  client: C,
```

Reply:
Oh, it works 😆